app/src/main/res/values/strings.xml

```diff
@@ -166,6 +166,8 @@
     <string name="title_terms_of_service">Terms of Service</string>
     <string name="key_terms_of_service" translatable="false">terms_of_service</string>
     <string name="legal">Legal</string>
+    <string name="visit_website">Visit Website</string>
+    <string name="key_visit_website">Visit Website</string>
```

Comment:
Change the value to visit_website

Reply:
@ShridharGoel can you be more specific?
First string is for text and second for it's key

Reply:
Yes so just change the value of the second string to 'visit_website' instead of 'Visit Website'